osi_sensorspecific.proto

```diff
@@ -6,6 +6,40 @@ import "osi_common.proto";
 
 package osi3;
 
+//
+// \brief Obstruction values for a given object and sensor.
+//
+// For a given detected moving object, and for a particular sensor, how
+// obstructed is that object from the point of view of that sensor.
+//
+// \note The ID of the sensor in question is available from the parent
+// SensorData.sensor_id field.  The ID of the object is available from
+// the parent DetectedMovingObject.DetectedItemHeader.ground_truth_id.
+//
+message Obstruction
```

Comment:
First question that comes to my mind (as a non-native speaker) is: Why is it named obstruction and not occlusion?